Makefile_v1

```diff
@@ -11,30 +11,47 @@ GOOGLEAPIS_REPO = $(REPO_DIR)/googleapis
 TESTS_REPO = $(REPO_DIR)/conformance-tests
 TEST_PROTO_DIR = $(TESTS_REPO)/firestore/v1
 TEST_PROTO_SRC = $(TEST_PROTO_DIR)/proto/google/cloud/conformance/firestore/v1/tests.proto
+TESTDATA_DIR = `pwd`/tests/unit/v1/testdata/
 
 TMPDIR = /tmp/python-fs-proto
-TMPDIR_FS = $(TMPDIR)/google/cloud/firestore_v1/proto
+TMPDIR_FS = $(TMPDIR)/google/cloud/firestore_v1/types
 TEST_PROTO_COPY = $(TMPDIR_FS)/tests.proto
```

Comment:
Hmm, I'm wondering if the `tests.proto`-derived module should live somewhere else, e.g., maybe in `tests/unit/V1`?  It's name is unfortunate, if we do that (maybe rename it to `conftests.proto`?

Reply:
I would be alright with either change, if only because we don't need to ship protos otherwise.

Reply:
I think I'll go ahead and move it.

Reply:
Hmm, the Docker version of the generator (`gcr.io/gapic-images/gapic-generator-python`) doesn't seem to find the `tests.proto` file if I move it out of `google/`.